packages/subgraph/tests/helpers/method-classes.ts

```diff
@@ -378,13 +378,17 @@ class ERC20ContractMethods extends ERC20Contract {
     if (!this.symbol) {
       throw new Error('Symbol is null');
     }
-    // we cast to string only for stoping rust compiler complaints.
-    createTokenCalls(
+    let supply = '10';
+    if (totalSupply) {
+      supply = totalSupply;
+    }
+    // we cast to string only for stoping rust compiler complaints.      createERC20TokenCalls(DAO_TOKEN_ADDRESS, '10', 'name', 'symbol', '6');
```

Comment:
Same here. Moreover, what is the purpose behind the `createERC20TokenCalls(DAO_TOKEN_ADDRESS, '10', 'name', 'symbol', '6');` comment?

Reply:
forgot to remove it 😅